ui/src/timeMachine/actions/queryBuilder.ts

```diff
@@ -413,28 +439,50 @@ export const selectTagValue = (index: number, value: string) => (
   }
 }
 
-export const selectBuilderFunction = (name: string) => (
+export const multiSelectBuilderFunction = (name: string) => (
   dispatch: Dispatch<Action>,
   getState: GetState
 ) => {
-  const state = getState()
-  const {
-    timeMachines: {activeTimeMachineID},
-  } = state
-  const {draftQueries, activeQueryIndex} = getActiveTimeMachine(state)
+  const {draftQueries, activeQueryIndex} = getActiveTimeMachine(getState())
 
   const functions = draftQueries[activeQueryIndex].builderConfig.functions
 
   let newFunctions: BuilderFunctionsType[]
 
   if (functions.find(f => f.name === name)) {
+    if (functions.length == 1) {
```

Comment:
doesn't this always return true since the find operation is being run before it?

Reply:
it doesn't, but it's confusing so I will assign some well named variables  :)